packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -128,6 +128,7 @@ export interface CheckoutState {
     isRedirecting: boolean;
     hasSelectedShippingOptions: boolean;
     isBuyNowCartEnabled: boolean;
+    isSubscribed: boolean;
```

Comment:
When the server-side implementation is done, we can get rid of everything in `checkout.tsx` from this PR.
Show the subscription choice of a shopper based on something like this:
```
const { isSubscribed } = getBillingAddress();
```